source/installation-guide/elasticsearch-cluster/elasticsearch-multi-node-cluster.rst

```diff
@@ -83,7 +83,7 @@ Elasticsearch installation and configuration
 Certificates creation and deployment
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-#. [*Master*] This step implies the selection of the Wazuh cluster installation type. In case of configuring a Wazuh single node cluster, the ``Wazuh single-node cluster`` tab must be selected. In case of having two or more nodes, the ``Wazuh multi-node cluster`` tab must be chosen.
+#. [*Master*] This step implies the selection of the Wazuh cluster installation type. Choose between ``Wazuh single-node cluster`` tab, for the Wazuh single node cluster, and ``Wazuh multi-node cluster`` tab in case of having two or more Wazuh cluster nodes.
```

Comment:
I think that we have redundancy here: 

> Choose between **Wazuh single-node cluster** tab, for the **Wazuh single node cluster**. 

Something like: Choose between **Wazuh single-node cluster**, for only one Wazuh server instance installation type.

Reply:
I will change.